viewer/packages/cad-styling/src/UnionNodeCollection.ts

```diff
@@ -36,4 +40,19 @@ export class UnionNodeCollection extends CombineNodeCollectionBase {
     }
     return set;
   }
+
+  public getAreas(): AreaCollection {
+    if (this._cachedNodeAreas) {
+      return this._cachedNodeAreas;
+    }
+
+    const newAreaCollection = new ClusteredAreaCollection();
+
+    for (let i = 1; i < this._nodeCollections.length; ++i) {
+      newAreaCollection.addAreas(this._nodeCollections[i].getAreas().areas());
+    }
```

Comment:
```suggestion
    for (let i = 0; i < this._nodeCollections.length; ++i) {
      newAreaCollection.addAreas(this._nodeCollections[i].getAreas().areas());
    }
```

Add unit test for this